dora/core/server/master/src/main/java/alluxio/master/job/DoraLoadJob.java

```diff
@@ -257,13 +200,14 @@ public List<DoraLoadTask> prepareNextTasks() {
     }
 
     Map<WorkerInfo, List<DoraLoadTask>> workerToTaskMap = new HashMap<>();
-    for (URIStatus uriStatus : batchBuilder.build()) {
+    for (UfsStatus ufsStatus : batchBuilder.build()) {
       // NOTE: active workers may not reflect all workers at start up,
-      // but hashbased policy will deterministicly only among current recognized active workers
-      WorkerInfo pickedWorker = mWorkerAssignPolicy.pickAWorker(uriStatus.getPath(),
+      // but hash based policy will deterministically only among current recognized active workers
+      WorkerInfo pickedWorker = mWorkerAssignPolicy.pickAWorker(
+          ufsStatus.getUfsFullPath().toString(),
```

Comment:
dora client actually uses alluxio path to hash to workers instead of ufs path, plz double confirm with others

Reply:
thanks i'll double check

Reply:
read the code and asked other folks, it is actually the ufs path.